i18next-scanner.config.js

```diff
@@ -0,0 +1,48 @@
+const { i18nextToPo } = require('i18next-conv');
+const VirtualFile = require('vinyl');
+
+module.exports = {
+  input: [
+    'src/**/*.{html,js}',
+  ],
+  options: {
+    contextSeparator: '|',
+    pluralSeparator: '|',
+    ns: [
+      'categories',
+      'footer',
+      'meal-planner',
+      'navigation',
+      'search',
+      'shopping-list',
+      'starred-recipes',
+    ],
+    resource: {
+      loadPath: 'public/locales/{{lng}}/{{ns}}.json',
+      savePath: 'i18n/locales/templates/{{ns}}.json',
```

Comment:
Thanks to the [custom flush method](https://github.com/openculinary/frontend/pull/65/files#diff-ad2bfd976f6361b8871f522adef0877aR41) defined later in this config file, the final filename will have a `.po` extension and the contents will be written in `gettext` format.

It would be nice if this functionality could be upstreamed into [i18next-scanner](https://github.com/i18next/i18next-scanner/) itself.